src/main/cpp/threadspecificdata.cpp

```diff
@@ -60,13 +60,16 @@ ThreadSpecificData& ThreadSpecificData::getDataNoThreads()
 
 ThreadSpecificData* ThreadSpecificData::getCurrentData()
 {
-#if APR_HAS_THREADS
 	void* pData = NULL;
+#if APR_HAS_THREADS
```

Comment:
should these `#if` be reversed, such that if it has `thread_local` that takes precedence over APR?